misc/wasmtime-rust/macro/src/lib.rs

```diff
@@ -64,6 +64,16 @@ fn generate_load(item: &syn::ItemTrait) -> syn::Result<TokenStream> {
                 ..Default::default()
             });
             let data = #root::wasmtime_interface_types::ModuleData::new(&bytes)?;
+            if let Some(wasi_name) = data.has_wasi() {
+                let wasi_handle = wasmtime_wasi::instantiate_wasi(
+                    "",
```

Comment:
Should this string be `wasi_name`?

I'm not actually sure what this string is...

Reply:
using `module_name` here